resources/helm-broker/templates/deploy.yaml

```diff
@@ -135,22 +113,19 @@ spec:
         # Temporary solution for readiness probe
         # Ref: https://github.com/istio/istio/issues/2628
         readinessProbe:
-          exec:
-            command:
-              - curl
-              - localhost:{{ .Values.ctrl.statusPort }}/ready
-          failureThreshold: 3
+          httpGet:
+            path: /ready
+            port: {{ .Values.ctrl.statusPort }}
           periodSeconds: 10
-          successThreshold: 1
           timeoutSeconds: 2
+          initialDelaySeconds: 5
```

Comment:
in other components you have `10`, maybe it's better to also increase here this delay?

Reply:
yes it should be set to 10 👍